### PR TITLE
Added displacement rates for all hazard classes. See #3553

### PR DESCRIPTION
**Pull request by timlinux**

### CODE REVIEW

**Comment by timlinux** [comment]
CC @lucernae and @Gustry

**Comment by Charlotte-Morgan** [comment]
Hi @timlinux @Gustry @lucernae 
its great to see displacement rates included in the hazard classes now.
From what I see, and ignoring earthquake which is special, its only flood classes that have a displacement rate calculated and that is back to the 1%. Does this mean that no other hazard classes / impact functions had a % applied to the 'affected' ?
Ie everyone in a hazard class with affected = true was 'affected' and had minimum needs calculated for them?

File: safe/definitions/hazard_classifications.py [hunk 1]
```diff
@@ -58,10 +58,10 @@
             'affected': True,
             'description': tr('The locations having the highest impact.'),
             'string_defaults': ['high'],
+            'displace_rate': 0.0,
```
Comment (timlinux) [inline safe/definitions/hazard_classifications.py]: displacement

Comment (ismailsunni) [inline safe/definitions/hazard_classifications.py]: Did you review your own PR @timlinux ? :)

**Comment by lucernae** [comment]
To reply @Charlotte-Morgan :
Minimum needs now calculated from displacement number. So if it is zero (for other hazard classification other than floods on population), all minimum needs number (also vulnerabilities and age) will be zero. Then I think we should hide minimum needs section if that's the case and have some explanatory text that no minimum needs section, since displacement number is zero.

**Comment by timlinux** [comment]
Great thanks @lucernae and yes lets put in placeholder text when there are minimum needs - something like:

> There is no minimum needs report displayed here because either
> a) there are no displaced people expected within the analysis area; or
> b) the hazard classes used for this analysis do not have displacement rates defined.

> If you believe this is an error, please submit an issue on the InaSAFE issue tracker at http://github.com/inasafe/inasafe/issues

How does that sound for you @Charlotte-Morgan

**Comment by timlinux** [comment]
@Charlotte-Morgan 

> From what I see, and ignoring earthquake which is special, its only flood classes that have a displacement rate calculated and that is back to the 1%. Does this mean that no other hazard classes / impact functions had a % applied to the 'affected' ?
Ie everyone in a hazard class with affected = true was 'affected' and had minimum needs calculated for them?

I am going to need to go and poke around in the old IF's to see how displacement rates were computed. I suspect a ratherblanket policy was applied but will need to review the old code before I can verify that.

File: safe/definitions/hazard_classifications.py (hunk 1 above)
Comment (timlinux) [inline safe/definitions/hazard_classifications.py]: I did indeed :-)

**Pull request merged**